### ISSUE TITLE
[Feature Request] iOS support

### ISSUE THREAD

**Issue by RenanDelfanti** [g123k/flutter_plugin_device_apps#21]
Any news about this feature?

**Comment by g123k** [comment]
Right now, it is strictly impossible, since Apple doesn't provide any API for this.

**Comment by RenanDelfanti** [comment]
@g123k  So I can't open another app inside my flutter app?

**Comment by g123k** [comment]
Yes you can, but the idea of this plugin is to be able to list all
installed apps. Not necessarily to launch them.

Le mar. 10 déc. 2019 à 06:41, Renan Delfanti <notifications@github.com> a
écrit :

**Comment by emvaized** [comment]
> Yes you can, but the idea of this plugin is to be able to list all installed apps. Not necessarily to launch them.

How about [this](https://stackoverflow.com/a/40148627/11381400) method?

**Comment by emvaized** [comment]
> Right now, it is strictly impossible, since Apple doesn't provide any API for this.

How about [this](https://stackoverflow.com/a/40148627/11381400) method?

**Comment by yourshinsuke** [comment]
https://pub.dev/packages/flutter_appavailability/install

**Comment by yourshinsuke** [comment]
it is better.

http://sanya.hzhbcjq.com/packages/installed_apps/example

**Comment by neophob** [comment]
Thanks for this library! Running on Android works like a charm. I tried to cross compile my app to iOS for testing - and while see the main issue I thought, hey I return a mocked list. But the Application constructors are private only - any reason why?

Let me know if you accept PR's for this case.

**Comment by g123k** [comment]
The problem will this kind of library on iOS: the validation from Apple.

You can of course use all the APIs you want (even the hidden ones from Apple), but they won't be accepted on the App Store.
That's the reason why this plugin doesn't implement such a feature.

**Comment by neophob** [comment]
Yeah, I see that point - my goal was to provide some help to simplify testing, thus provide a mocked app list. But for this I would need to create a new Application object

**Comment by muhd-ameen** [comment]
Any updates on this feature? @g123k
